code/modules/events/radiation.dm

```diff
@@ -19,9 +19,11 @@
 		for (var/pulses = pulse_amt, pulses > 0, pulses--)
 			pulseloc = pick(wormholeturfs)
 			pulse_lifespan = rand(min_pulse_lifespan,max_pulse_lifespan)
-			new /obj/anomaly/radioactive_burst(pulseloc,lifespan = pulse_lifespan)
+			pick(prob(100); new /obj/anomaly/radioactive_burst(pulseloc,lifespan = pulse_lifespan), prob(50); new /obj/anomaly/neutron_burst(pulseloc,lifespan = pulse_lifespan))
```

Comment:
prob100 is useless

Reply:
tell that to the miracle matter proc lol